### ISSUE TITLE
Can't load developer applet in Das Keyboard Q on Ubuntu

### ISSUE THREAD

**Issue by factorypreset** [daskeyboard/daskeyboard.io#91]
From URL: https://www.daskeyboard.io/applet-development/index.md

At present there's a bug in the way dev applets are loaded into the Das Keyboard Q environment. To replicate:

 * ubuntu 20.04 LTS
 * Das Keyboard Q version 3.2.7
 * Das Keyboard 5Q hardware
 * Firmware v7.4.51
 * clone any applet from github, e.g. https://github.com/daskeyboard/daskeyboard-applet--montastic
 * in Das Keyboard Q, click settings icon and toggle dev menu on
 * in dev menu (`</a>`) click 'load applet'
 * in the 'Create dev applet' menu, click the 'select' button. This will load a file manager (looks like nautilus)
 * navigate to where you cloned the repo, and highlight the directory
 * click 'select'
 * Das Keyboard Q fails to select the folder, but instead picks a file within that folder (in my experience it always picks `README.md`)
 * This appears in `applet.log.json`:

```
{"level":"info","message":"2020-12-10 02:35:56.3556 Install request:","timestamp":"2020-12-10T19:35:56.282Z"}
{"level":"info","message":"2020-12-10 02:35:56.3556 Installing appletId: -279","timestamp":"2020-12-10T19:35:56.283Z"}
{"level":"info","message":"2020-12-10 02:35:56.3556 Installing from: file:///home/bo/src/daskeyboard-applet--montastic/README.md","timestamp":"2020-12-10T19:35:56.283Z"}
{"level":"info","message":"2020-12-10 02:35:56.3556 Installing with user config: ","timestamp":"2020-12-10T19:35:56.283Z"}
{"level":"info","message":"2020-12-10 02:35:56.3556 Beginning installation of applet: -279, release: file:///home/bo/src/daskeyboard-applet--montastic/README.md","timestamp":"2020-12-10T19:35:56.286Z"}
{"level":"info","message":"2020-12-10 02:35:56.3556 Installing applet at: file:///home/bo/src/daskeyboard-applet--montastic/README.md","timestamp":"2020-12-10T19:35:56.287Z"}
{"level":"info","message":"2020-12-10 02:35:56.3556 Sideloading from path: /home/bo/src/daskeyboard-applet--montastic/README.md to /home/bo/.quio/v2/q_extensions/-279","timestamp":"2020-12-10T19:35:56.287Z"}
{"level":"error","message":"2020-12-10 02:35:56.3556 Error: ENOTDIR: not a directory, open '/home/bo/.quio/v2/q_extensions/-279/package.json'","timestamp":"2020-12-10T19:35:56.295Z"}
{"level":"error","message":"2020-12-10 02:35:56.3556 Error installing package at /home/bo/.quio/v2/q_extensions/-279; Error reading file at '/home/bo/.quio/v2/q_extensions/-279/package.json'.","timestamp":"2020-12-10T19:35:56.296Z"}
{"level":"error","message":"2020-12-10 02:35:56.3556 Error while installing applet -279","timestamp":"2020-12-10T19:35:56.296Z"}

```

Presumably this indicates some incorrect interactions between the file manager (nautilus) and the Das Keyboard Q software.

Is there a way to bypass the file manager GUI and set the value directly? Can I otherwise trick the software into loading a directory instead of a single file?

**Comment by fabiendv** [comment]
Good morning,

Thank you for opening an issue. 
Our team will have a look soon.

**Comment by Mubramaj** [comment]
Hi,

This issue should be fixed in version 3.2.8 of the software.

Thank you for reporting it.
